### ISSUE TITLE
Effect of :training() method function?

### ISSUE THREAD

**Issue by ArturoDeza** [edenton/drnet#6]
Hi Emily,
Perhaps this is a more general question also overlapping with Torch, but what is the main function of using the :training() method function for the Context and Pose encoder networks? In previous torch code I have seen, I don't usually see this method being called explicitly when training a network, so perhaps I am missing something? Does adding the :training() method unfreeze weights? Is it an initializer (it doesn't seem like one). I also can't find the main source file for it.
Arturo

**Comment by edenton** [comment]
:training() and :evaluate() puts the networks in training/testing mode, which has an effect for certain modules.. For example, dropout and batchnorm have distinct cases depending on if the model is being trained or used at eval time.
